swagger.js

```diff
@@ -31,6 +32,104 @@ const doc = {
         },
       },
     },
+    UserWithAlbums: {
+      $id: '64d6ca92a15d2e18ab96a2a3',
+      $name: 'John Doe',
+      $username: 'johndoe',
+      $email: 'john@example.com',
+      $role: { '@enum': ['user', 'admin'] },
+      profileImage: {
+        url: 'https://example.com/profile-image.jpg',
+      },
+      creditCardInfo: {
+        hashedNumber: '1234567890',
+        expiry: '12/24',
+        preferredPaymentOption: {
+          '@enum': ['credit card', 'paypal', 'google pay'],
+        },
+      },
+      $purchasedAlbums: [
+        {
+          $_id: '64ef50c8c5551074444547bc',
+          $user: '64d6ca92a15d2e18ab96a2a3',
+          $album: {
+            $id: '64d2a94c793389a43fc5a8d6',
+            $albumName: '...Baby one more time',
+            $artistName: 'Britney Spears',
+            $createdAt: '2023-08-08T20:45:00.942Z',
+            $updatedAt: '2023-08-17T10:10:42.861Z',
+            $price: 20.99,
+            image:
+              'https://i.scdn.co/image/ab67616d0000b27371cae34ad5a39bdab78af13e',
+            spotifyUrl:
+              'https://api.spotify.com/v1/albums/6r1lh7fHMB499vGKtIyJLy',
+            releaseDate: '2023-07-28T00:00:00.000Z',
+            averageRating: 0,
+            numOfReviews: 0,
+          },
+        },
+      ],
+    },
+    Album: {
+      $id: '64d2ae2308a725b72bd5c0cf',
+      $albumName: 'The Dark Side of the Moon',
+      $artistName: 'Pink Floyd',
+      price: 9.99,
+      image: '/uploads/example.jpg',
+      releaseDate: '2022-03-01T00:00:00.000Z',
```

Comment:
also comes with `createdAt` and `updatedAt` fields:

```suggestion
      releaseDate: '2022-03-01T00:00:00.000Z',
      createdAt: '2022-03-01T00:00:00.000Z',
      updatedAt: '2022-03-01T00:00:00.000Z',
```